### PR TITLE
Added cart page

### PR DESCRIPTION
**Pull request by nishasen**
image

### CODE REVIEW

File: Pages/CartManagement/cart.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                        <button class="btn btn-lg icon-lg">
+                            <a href="/Pages/CartManagement/cart.html" class="btn-link text-primary hover-primary">
+                                <i class="fa fa-shopping-cart"></i>
+                            </a>
+                        </button>
+                        <span class="badge-number secondary-bgcolor" id="cart-items">0</span>
+                    </div>
+                    <button class="btn btn-lg icon-lg btn-user">
+                        <a href="" class="btn-link text-primary hover-primary">
+                            <i class="fa fa-user-circle-o"></i>
```
Comment (shuntzhere) [inline Pages/CartManagement/cart.html]: Here is a blank href.

File: Pages/CartManagement/cart.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                        </a>
+                        <div class="dropdown-content">
+                            <a href="Pages/Authentication/login.html" class="btn-link">Login</a>
+                            <a href="" class="btn-link">User Profile</a>
+                            <a href="" class="btn-link">Logout</a>
+                        </div>
```
Comment (shuntzhere) [inline Pages/CartManagement/cart.html]: Blank href's

File: Pages/CartManagement/cart.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                </div>
+                <div class="product centered product-ravenclaw product-houses">
+                    <img src="/Assets/Ravenclaw.png" class="img-responsive">
+                </div>
+                <div class="product centered product-hufflepuff product-houses">
+                    <img src="/Assets/Hufflepuff.png" class="img-responsive">
+                </div>
+                <div class="product centered product-slytherin product-houses">
+                    <img src="/Assets/Slytherin.png" class="img-responsive">
+                </div>
```
Comment (shuntzhere) [inline Pages/CartManagement/cart.html]: Give images an alt text.

File: Pages/CartManagement/cart.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+                            </button>
+                        </div>
+                    </div>   
+                </div>
+                <div class="cart-div centered">
+                    <div class="cart-card">
+                        <div class="items">
+                            <span class="badge-text badge-box-square fire-bgcolor badge-over-image">49.99%</span>
+                            <img src="/Assets/Gryffindor-tshirt.png" class="product-image img-responsive">
+                        </div>
```
Comment (shuntzhere) [inline Pages/CartManagement/cart.html]: Add alt text to an image.

File: Stylesheets/PageStyles/cart.css [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+     display: flex;
+     justify-content: center;
+     flex-wrap: wrap;
+     gap: 2rem;
+     z-index: 5;
+     margin-top: 1rem;
+}
+
+.cart-div {
+    /* grid-area: 1/1/2/2; */
```
Comment (shuntzhere) [inline Stylesheets/PageStyles/cart.css]: Make sure to remove the commented out code before the merge.

File: Pages/CartManagement/cart.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+                    <div class="badge-icon badge-wrapper">
+                        <button class="btn btn-lg icon-lg">
+                            <a href="/Pages/CartManagement/cart.html" class="btn-link text-primary hover-primary">
+                                <i class="fa fa-shopping-cart"></i>
+                            </a>
+                        </button>
+                        <span class="badge-number secondary-bgcolor" id="cart-items">0</span>
+                    </div>
+                    <button class="btn btn-lg icon-lg btn-user">
+                        <a href="" class="btn-link text-primary hover-primary">
```
Comment (shubhamkumar648) [inline Pages/CartManagement/cart.html]: dont keep href tag empty use #

**Review by shuntzhere — CHANGES_REQUESTED** [review]
The code looks good, although suggested a few changes.
Also if you are using your component library, you can use utility classes to remove the extra lines of code that keep on repeating.

**Review by shubhamkumar648 — APPROVED** [review]
put alt attributes  in all image tags, use color value in var(), great UI design rather than that code looks clean, you can merge  after modifying the code

File: Pages/CartManagement/cart.html (hunk 1 above)
Comment (nishasen) [reply Pages/CartManagement/cart.html]: I will change this

File: Pages/CartManagement/cart.html (hunk 2 above)
Comment (nishasen) [reply Pages/CartManagement/cart.html]: Yes will change these

**Review by shuntzhere — APPROVED** [review]
Looks really good. You can go ahead with the merge.

**Pull request merged**